Configuration/PyReleaseValidation/scripts/test-MC-setup_hlt.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+############################################################################################################
+### HLT STEP in a different release
+
+function rel_setup()
+{
+    scram p CMSSW $1
```

Comment:
out of curiosity, since this is "fixed" instead of creating a new checkout area would it not be possible to just do `cmsenv` inside the area `/cvmfs/cms.cern.ch/$hlt_arch/cms/cmssw/$1/` ?

Reply:
I think so, actually seems cleaner too. Let me try.